maintainers/maintainer-list.nix

```diff
@@ -6303,12 +6303,6 @@
     githubId = 1843676;
     name = "Jonathan Reeve";
   };
-  jonnybolton = {
```

Comment:
Split this commit into two.

If you need some tutorial, read this SO post:

https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits